.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,45 @@
+name: "CodeQL"
+
+on:
+  # TODO(clason): remove when done testing
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '42 0 * * *'
```

Daily is a bit much, let's go with weekly.

```suggestion
    - cron: '42 0 * * 0'
```